runtime/gc_base/GCExtensions.cpp

```diff
@@ -235,11 +235,9 @@ MM_GCExtensions::updateIdentityHashDataForSaltIndex(UDATA index)
 }
 
 void
-MM_GCExtensions::computeDefaultMaxHeap(MM_EnvironmentBase *env)
+MM_GCExtensions::computeDefaultMaxHeapForJava(bool eanbleOriginalJDK8HeapSizeCompatibilityOption)
 {
-	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
-
-	MM_GCExtensionsBase::computeDefaultMaxHeap(env);
+	OMRPORT_ACCESS_FROM_OMRVM(_omrVM);
```

Comment:
I'm fine with removing the call to EnvBase::computeDefaultMaxHeap here. Just commenting for clarification purposes - it will still be called during EnvBase::initialize.

Reply:
done